staging/src/k8s.io/pod-security-admission/policy/check_allowPrivilegeEscalation.go

```diff
@@ -18,7 +18,6 @@ package policy
 
 import (
 	"fmt"
-
```

Comment:
nit: keep this
(kubernetes import style has 3 blocks: go stdlib, kubernetes imports, and other 3rd party packages)